marlowe-actus/app/Main.hs

```diff
@@ -0,0 +1,180 @@
+{-# LANGUAGE GADTs #-}
+{-# LANGUAGE QuasiQuotes #-}
+{-# LANGUAGE ScopedTypeVariables #-}
+{-# LANGUAGE ViewPatterns #-}
+
+module Main where
+
+import Data.Int (Int32)
+import qualified Language.R as R
+import Language.R (R)
+import Language.R.QQ
+import Language.Marlowe.ACTUS.Analysis(sampleCashflows)
+import Data.Time.Calendar(showGregorian, fromGregorian)
+import Language.Marlowe.ACTUS.Definitions.ContractTerms
+import Language.Marlowe.ACTUS.Definitions.BusinessEvents(RiskFactors(..))
+import Language.Marlowe.ACTUS.Definitions.Schedule(CashFlow(..))
+import Data.Aeson(encode, decode)
+import Data.String (IsString (fromString))
+import Data.ByteString.Lazy.Char8(unpack)
+
```

Comment:
Don't forget to run fix-stylish-haskell. I can see it's was not run here :)

Reply:
project won't pass checks anyway (I hope!)